docs/Developer's Guide.md

```diff
@@ -125,24 +125,20 @@ Sometimes, you would like to connect the frontend on your local computer to the
 
 2. Create the SSH tunnel
 
-   After setting up the ssh server on your host machine, find your host IP or DNS address. Open a command prompt or terminal window and enter the following:
+   After setting up the ssh server on your local machine, find your host IP or DNS address.  Open a command prompt or terminal window and enter `ipconfig` (Windows), `hostname` and/or `ifconfig` (MacOS), or `hostname -I` and/or `hostname -f` (Linux).  You will need to know this so you can connect to your local machine from the virtual machine.
+   
+   Log into the VM and open a Git Bash shell or a command prompt/powershell window. Enter the following command:
```

Comment:
`hostname` works on Windows as well. That is why I wrote hostname for all of them. I would rather not confuse people with a bunch of different cases if we don't have to.

Reply:
Okay.  Further down the instructions said to find the IP number or hostname using ipconfig on Windows, ifconfig on mac and linux so I just expanded that and moved it up.  On linux ifconfig has been replaced by ip (which I don't like nearly as well as the output is not as clean).  Also the hostname command on Linux may or may not return the FQDN (depending on how the host name was set), so the "-f" option is helpful.  Together this just means that "hostname" by itself does not work for all platforms.
I'll make some adjustments to the doc.

Reply:
I've tried commands on Windows and Linux, as well as tried the hostname and ifconfig commands on MacOS.  I think some more detail is required than that provided by just hostname on the various platforms since we need an FQDN or IP number.  The latest revisions are even a bit longer but hopefully clearer.